packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -337,7 +337,17 @@ export const makeZCFZygote = (
 
   let contractCode;
 
-  /** @type {ZCFZygote} */
+  /**
+   * A zygote is a pre-image of a vat that can quickly be instantiated because
```

Comment:
Please have @warner check this use of the term "pre-image".
@warner , waddaya think?

Reply:
yeah, that works